src/rgw/rgw_lua_utils.h

```diff
@@ -0,0 +1,175 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <ctime>
+#include <lua.hpp>
+
+class CephContext;
+
+namespace rgw::lua {
+
+// push ceph time in string format: "%Y-%m-%d %H:%M:%S"
+template <typename CephTime>
+void lua_pushtime(lua_State* L, const CephTime& tp) 
```

Comment:
These should not be in the `lua_` namespace (even if they are C++ procedures with different names).

Reply:
makes sense.
should also change the `lua_...` function in the cc file.
I would go for `pushtime()`, `pushstring()` etc. since they are inside the `rgw::lua` C++ namespace anyway.
however, as this PR is already merged, can you add that to: https://github.com/ceph/ceph/pull/37657

Reply:
done